sdk/eventgrid/ci.yml

```diff
@@ -1,5 +1,3 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
@dw511214992 Do you have any idea why this line will get removed ? I have change the new generate ci.yml logic but not idea about this.